.github/workflows/release.sh

```diff
@@ -107,18 +107,25 @@ echo "::endgroup::"
 pick_otp_vsn() {
     global_OTP_VSN=undefined
     while read -r release; do
-        prepare_git_tag "${release}"
-
-        pushd "${global_INITIAL_DIR}" || exit
-        if grep "${global_GIT_TAG} " _RELEASES; then
-            continue
+        if [[ $release =~ ^[0-9].*$ ]]; then
+            major=${release%%.*}
+            if [[ $major -lt 24 ]]; then
+                continue
+            fi
+
+            prepare_git_tag "${release}"
+
+            pushd "${global_INITIAL_DIR}" || exit
+            if test -f _RELEASES && grep "${global_GIT_TAG} " _RELEASES; then
+                continue
+            fi
+            popd || exit
+
+            global_OTP_VSN=${release}
+            break
         fi
-        popd || exit
-
-        global_OTP_VSN=${release}
-        break
-    done < <(./kerl update releases | tail -n 70)
-    if [[ "${global_OTP_VSN}" == "undefined" ]]; then
+    done < <(./kerl list releases all)
```

Comment:
I broke this in `kerl` recently 😄 and didn't even remember we depended on this command.